GitUI/CommitInfo/CommitInfo.cs

```diff
@@ -213,7 +213,7 @@ void UpdateCommitMessage()
                 }
 
                 var commitMessage = _commitDataBodyRenderer.Render(data, showRevisionsAsLinks: CommandClickedEvent != null);
-                rtbxCommitMessage.SetXHTMLText(commitMessage);
+                rtbxCommitMessage.SetXHTMLText(commitMessage + Environment.NewLine);
```

Comment:
I suggest a comment about the workaround here (to avoid Blame->PR discussion to find why).

Reply:
Good point, will do

Reply:
I actually was planning to merge your branch as is, unsquashed. 
This is one of those rare instances where I want to preserve the full history, given that we've (you mainly) gone through getting it right.

Reply:
ok, let me unsquash then